my-app/src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react'
+import "./Homepage.css"
+import Nav from '../../components/Nav/Nav'
+import hero_img from "../../assets/home_img.jpg"
+import { Link } from 'react-router-dom'
+import ViCard from '../../components/VideoCard/ViCard'
+import axios from "axios"
+
+
+const Homepage = () => {
+  const [showVideoList, setShowVideoList] = useState([]);
+  useEffect(() => {  
+  (async () =>{
+   try {
+     const resp = await axios.get(`/api/videos`)
+     setShowVideoList(resp.data.videos)
+     console.log(resp.data.videos)
+   } catch (error) {
+     console.log(error)
```

Comment:
nishant suggested checking type of error and throwing relevant toast. 
I havent done this as well, just a suggestion for the future.

Reply:
didnt have toast for that as of now. but moving forward . will add it